### PR TITLE
Add Moq and NSubstitute

### PR DESCRIPTION
**Pull request by dtchepak**
Tried adding this after @blairconrad tagged me in #194. 😂
@stakx please check you are ok with the Moq entries. 😄

### CODE REVIEW

**Comment by blairconrad** [comment]
Oh, I should've just added them myself. But I was afraid it was too pushy and/or I'd mess stuff up. Then I toyed with the idea of giving you guys write access to my fork so you could fix my mistakes. Then I figured it was too complicated and y'all might as well do your own.

File: input/data/projects/nsubstitute.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - NSubstitute.Analyzers.VisualBasic
+Language: C#
+Image: http://nsubstitute.github.io/images/nsubstitute-100x100.png
+Title: NSubstitute
+Description: NSubstitute is a friendly substitute for .NET mocking frameworks. It has a simple, succinct syntax to help developers write clearer tests. NSubstitute is designed for Arrange-Act-Assert (AAA) testing and with Test Driven Development (TDD) in mind.
+Website: https://nsubstitute.github.io/
+Docs: https://nsubstitute.github.io/help.html
+Twitter: nsubstitute
+Tags:
+  - Testing
```
Comment (blairconrad) [inline input/data/projects/nsubstitute.yml]: Chat:L

Comment (dtchepak) [reply input/data/projects/nsubstitute.yml]: I'm not sure that counts as chat? I'm thinking of moving away from the group in favour of GH issues and StackOverflow anyway, as that seems to be where most people tend to try first.

**Comment by dtchepak** [comment]
@blairconrad All good. 👍
I'm good at doing the "but what if..." thing as well. 😂

File: input/data/projects/nsubstitute.yml (hunk 1 above)
Comment (blairconrad) [reply input/data/projects/nsubstitute.yml]: Oh. Sorry. Yeah, sometimes Gitter/GitHub/StackOverflow can be a bit much.

File: input/data/projects/moq.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Source: https://github.com/moq/moq4
+NuGet:
+  - Moq
+Language: C#
+Image: https://raw.githubusercontent.com/moq/moq4/master/moq-bigger.png
+Title: Moq
+Description: Moq is the most popular and friendly mocking framework for .NET
+Website: https://github.com/moq/moq4
+Docs: http://www.nudoq.org/#!/Projects/Moq
```
[the remark this answers is not in the document]
Comment (dtchepak) [reply input/data/projects/moq.yml]: AFAICT [only one doc is supported](https://github.com/daveaglick/discoverdotnet/blob/d05a48903b5e3a0fd9ef89b19b1e1b9cff451f85/input/projects/_Layout.cshtml#L167). I'll update it to use the Quickstart for now 👍

**Comment by daveaglick** [comment]
@blairconrad 
> Oh, I should've just added them myself. But I was afraid it was too pushy and/or I'd mess stuff up.

Don't worry about that! More than happy to help folks get through submitting PRs, and the more projects indexed, the better.

**Comment by daveaglick** [comment]
Looks perfect - thanks all! Feel free to revise the files as the projects evolve.

**Pull request merged**